bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -30,13 +38,21 @@
 		<label>Siren</label>
 		<description>Status of the siren</description>
 		<category>Soundvolume</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>SoundVolume</tag>
+		</tags>
```

Comment:
I have the strange feeling that "Alarm" would be more appropriate.

Reply:
Is my assumption right that with this channel one controles the sound from the siren en not the alarm itself?
 E.g. there might also be flash lights or other ways to alert people of an alarm, er better there might be a silent alarm? If so this controls the sound and  not the alarm.

Reply:
That is how I would understand it. Perhaps it is to manually mute the siren after an alarm has occurred. Or something like that.